src/tools/generate/generate_sitemap.js

```diff
@@ -57,7 +57,9 @@ const get_static_pages = async (sitemap_languages) => {
     .reduce((x, y) => [...x, ...y], []);
 
   // Get the sitemap entries for those pages
-  let urls = [];
+  let urls = [
+    {url: 'en/2020/stories/page-content.html', lastmod: '2020-12-19'},
+  ];
```

Comment:
Revert this back:

```suggestion
  let urls = [];
```

And instead add this to line 7:

```js
const static_pages = [
  'index.html',
  'table_of_contents.html',
  'methodology.html',
  'contributors.html',
  'accessibility_statement.html',
  'stories/page-content.html'
];
```